bus_frequency/src/gtfs.rs

```diff
@@ -0,0 +1,151 @@
+use std::collections::BTreeMap;
+
+use anyhow::Result;
+use chrono::NaiveTime;
+use fs_err::File;
+use serde::Deserialize;
+
+// See https://gtfs.org/schedule/reference/. This groups all arrival_times per day and stop, with
+// some caveats:
+//
+// - Ignoring calendar_dates.txt, which lists days an extra service is run or is skipped
```

Comment:
It'd be good to fix all of these, but cost/benefit's not there right now. This layer will replace a manual process of looking at timetables on Streetview, which was not likely working well for spotting exceptions to regular service anyway. The decision making on top of this layer will pretty much be "are there lots of buses during a peak hour and is there near a scheme?"